src/main/java/uk/gov/hmcts/reform/slc/services/steps/getpdf/IHtmlToPdfConverter.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.hmcts.reform.slc.services.steps.getpdf;
+
+import java.util.Map;
+import java.util.function.BiFunction;
+
+public interface IHtmlToPdfConverter extends BiFunction<byte[], Map<String, Object>, byte[]> {
```

Comment:
Friendly alias for lambda instance has its benefits. Alternatively, to keep inheritance hierarchy flat and open to other implementations, there are options like:

1.
```java

interface IHtmlToPdfConverter {
    byte[] convert(byte[] template, Map<String, Object> values);
}

class HtmlToPdfFuncAdapter implements IHtmlToPdfConverter {

    private final BiFunction<byte[], Map<String, Object>, byte[]> function;

    HtmlToPdfFuncAdapter(BiFunction<byte[], Map<String, Object>, byte[]> function) {
        this.function = function;
    }

    @Override
    public byte[] convert(byte[] template, Map<String, Object> values) {
        return function.apply(template, values);
    }
}
```
2. Use `BiFunction<byte[], Map<String, Object>, byte[]>` as it is quite technical and descriptive.